docs/content/language-cheatsheet.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Language Cheatsheet
+navtitle: Language Cheatsheet
+kind: documentation
+weight: 5
+toc: true
+---
+
+## Assignment and Equality
+
+```ruby
+# assign variable x to value of field foo.bar.baz in input
+x := input.foo.bar.baz
+
+# check if variable x has same value as variable y
+x == y
+
+# check if variable x is a set containing "foo" and "bar"
+x == {"foo", "bar"}
+
+# OR
+
+{"foo", "bar"} == x
+```
+
+## Lookup
+
+**Arrays**
```

Comment:
Not sure if it really matters, but if you swap these to h4 or h5's (ala `#### Arrays` or `##### Arrays`) it should look very similar but they'll get anchor links slapped on 'em automatically. The h3 (###) makes the most sense from a markdown point of view since they are under a h2, but its quite a bit larger than the "strong" text.

Reply:
h4 looks good and doesn't show up in the sidenav. Will switch to that. Thanks!